clients/admin-ui/cypress/e2e/systems-plus.cy.ts

```diff
@@ -465,6 +466,20 @@ describe("System management with Plus features", () => {
     });
   });
 
+  describe("asset list", () => {
+    beforeEach(() => {
+      stubWebsiteMonitor();
+      cy.visit(`${SYSTEM_ROUTE}/configure/demo_analytics_system`);
+      cy.wait("@getSystem");
+    });
+
+    it("lists assets in the system assets tab", () => {
+      cy.getByTestId("tab-Assets").click({ force: true });
+      cy.wait("@getSystemAssets");
+      cy.getByTestId("row-0-col-name").should("contain", "ar_debug");
+    });
```

Comment:
Would love to see this pushed further: consider adding more assertions for other fields beyond just the name. Maybe test error states and empty states. Maybe even the pagination if possible.